### ISSUE TITLE
Column names in queries should be quoted

### ISSUE THREAD

**Issue by dandanlen** [diffix/explorer#154]
Column names should be quoted in case there are invalid characters in the column name.

Example:

```
select min(NumberOfTime60-89DaysPastDueNotWorse)
from loans_int
```

should be:

```
select min("NumberOfTime60-89DaysPastDueNotWorse")
from loans_int
```

**Comment by AndreiBozantan** [comment]
@sebastian Is it enough to just quote the names, or should we also use some kind of escaping?

**Comment by sebastian** [comment]
Quote only for now.

**Comment by AndreiBozantan** [comment]
solved with #238
